module_1/gradle/multi-project/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
```

Comment:
try to move java plugin statement inside of allProject and remove duplicates from other gradle.build files